pkg/monitor/monitor/monitor_manager.go

```diff
@@ -254,6 +255,25 @@ func (mm *MonitorManager) syncTidbMonitorConfig(tc *v1alpha1.TidbCluster, monito
 	if err != nil {
 		return nil, err
 	}
+	if monitor.Spec.Prometheus.Config != nil && monitor.Spec.Prometheus.Config.ConfigMapRef != nil && len(monitor.Spec.Prometheus.Config.ConfigMapRef.Name) > 0 {
```

Comment:
I think we can define a variable here like ` promConfig := monitor.Spec.Prometheus.Config` and use this variable instead of ` monitor.Spec.Prometheus.Config ` in following code.

Reply:
done